pandas/tseries/resample.py

```diff
@@ -208,6 +208,12 @@ def _assure_grouper(self):
         """ make sure that we are creating our binner & grouper """
         self._set_binner()
 
+    def plot(self, *args, **kwargs):
+        # for compat with prior versions, we want to
+        # have the warnings shown here and just have this work
+        return _maybe_process_deprecations(self, how='mean').plot(*args,
```

Comment:
Yep, that is what I mean (a more meaningful warning). As this is now rather confusing to the user IMO, as the warning says something about `how` while the user has not explicitely used it (before this PR, you got a "resample is now a deferred operation, please use `df.resample().mean() instead` message)